src/routes/embedded/account/add-wallet/account-add-wallet.view.tsx

```diff
@@ -0,0 +1,16 @@
+import { Link } from "~wallets/router/components/link/Link";
+
+export function AccountAddWalletEmbeddedView() {
+  return (
+    <div>
+      <h3>Add Wallet</h3>
```

Comment:
@Jonathansoufer There's also `AuthAddWalletEmbeddedView` which is almost identical but has different copy.